pkg/genlib/generator_with_text_template.go

```diff
@@ -37,6 +64,15 @@ func NewGeneratorWithTextTemplate(tpl []byte, cfg Config, fields Fields) (*Gener
 		return time.Duration(duration)
 	}
 
+	templateFns["awsAZFromRegion"] = func(region string) string {
```

Comment:
let's not forget to document this function

Reply:
I was wondering how/where do document these functions. I will follow-up with a PR adding it.